### DISCUSSION TITLE
Photo Side by Side and resizing

### DISCUSSION THREAD

**Discussion by WilliamBlacksmith** [cu-esiil-edu/2024-data-short-course#67] [Ideas]
Here is the code i used to get my photos side by side and resized.

<div>

    <table>
        <td> 
            <img src="Add photo link here" width="350"/>
        </td>
        <td>
            <img src="Add photo link here" width="275"/>
        </td>
    </table>

</div>

I used a table for the photo's but i had issues with one of the photos not showing up.  I clicked inspect on the page and my photo that wasnt' showing up had a different html link. so i deleted the phtoto and readded it. After that my photo showed up. Also i added <div> at the top of the code and </div> at the bottom of the code.

**Comment by nquarder** [comment]
Nice job troubleshooting @WilliamBlacksmith !

Your page is looking great, btw: https://williamblacksmith.github.io/ 👍
